web/src/components/Map/index.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { FunctionComponent, useEffect, useState } from 'react'
+import ReactMapGL from 'react-map-gl'
+// import styled from 'theme'
+// import MapFilters from './ControlPanel'
+import Pin from './Pin'
+import { Pollutant, PollutantValueResponse, Viewport } from './types'
+import {
+  MAPBOX_ACCESS_TOKEN,
+  MAP_STYLE,
+  POLLUTANTS_API_URL
+} from '../../constants'
+
+// const MapContainer = styled.div`
+//   position: relative;
+//   margin-top: 40px;
+// `
+
+const initialViewport: Viewport = {
+  zoom: 14,
+  latitude: 37.812036,
+  longitude: -122.286675,
+  bearing: 0,
+  pitch: 0
+}
+
+const parsePollutant = (item: PollutantValueResponse): Pollutant => {
+  const timeGeoSplit = item.time_geo.split('(')
+  const coordsSplit = timeGeoSplit[1].split(', ')
+  return {
+    timestamp: timeGeoSplit[0].trim(),
+    latitude: Number(coordsSplit[0].trim()),
```

Comment:
The `latitude` and `longitude` should be flipped here since this incorporates my bugfix from `a53912a` (longitude is at index 0 and latitude is at index 1)